src/util/util.js

```diff
@@ -26,10 +26,16 @@ export function cantorPair(x, y) {
   return ((x + y) * (x + y + 1)) / 2 + y
 }
 
-export function asHex(value, padding) {
-  if (value == null) return ''
-  if (value < 0) return value.toString()
-  return '0x' + value.toString(16).padStart(padding, '0').toUpperCase()
+export function asHex(input, padding) {
+  if (input == null) return ''
+  if (input < 0) return input.toString()
```

Comment:
Worth a comment that the `<` will trigger a ToNumber conversion on `input`, so this is fine even if input is a string.

Reply:
@jingteng25742 ^^^ Please note this comment from Boris. I am not 100% certain what you indented here, but I know we have occasional bug around showing things like `0X0x0104` and weird stuff like that.

I'll merge this in regardless for now, since this is not making things much worse, but please do check that.